File: python/uri1061.py
dia = input().split()
dia = int(dia[1])

tempo = input().split(":")
hour, minu, sec = list(map(int, tempo))

diaFinal = input().split()
diaFinal = int(diaFinal[1])

tempoF = input().split(":")
hourF, minF, secF = list(map(int, tempoF))

secResult = secF - sec
minResult = minF - minu
hourResult = hourF - hour
diaResult = diaFinal - dia

if secResult < 0:
    minResult = minResult - 1
    secResult = secResult + 60

if minResult < 0:
    hourResult = hourResult - 1
    minResult = minResult + 60

if hourResult < 0:
    diaResult = diaResult - 1
    hourResult = hourResult + 24

print(f"{diaResult} dia(s)")
print(f"{hourResult} hora(s)")
print(f"{minResult} minuto(s)")
print(f"{secResult} segundo(s)")      

File: c++/uri1061.cpp
#include <iostream>

using namespace std;

int main() {
    int diai, diaf, hi, mi, si, hf, mf, sf;
    char dia[4], s; //s de simbolo para o ':'
    
    cin >> dia >> diai >> hi >> s >> mi >> s >> si;
    cin >> dia >> diaf >> hf >> s >> mf >> s >> sf;

    int sResult = sf - si;
    int mResult = mf - mi;
    int hResult = hf - hi;
    int diaResult = diaf - diai;

    if (sResult < 0) {
        mResult = mResult - 1;
        sResult = sResult + 60;
    }
    if (mResult < 0) {
        hResult = hResult - 1;
        mResult = mResult + 60;
    }
    if (hResult < 0) {
        diaResult = diaResult - 1;
        hResult = hResult + 24;
    }
    
    cout << diaResult << " dia(s)\n";
    cout << hResult << " hora(s)\n";
    cout << mResult << " minuto(s)\n";
    cout << sResult << " segundo(s)\n";

    return 0;
}